feedback

pkumar999 opened the pull request.

caraz22 committed:
Added microservices assignment 7 as our project

caraz22 committed:
added hidden folders

AvaMcintosh committed:
Update Pet.java

Adding Pet Model File

AvaMcintosh committed:
Update PetDTO.java

Adding completed PetDTO

AvaMcintosh committed:
Update PetController.java

I have errors with my get method

AvaMcintosh committed:
Update PetRepository.java

Wasn't sure how to do the query

AvaMcintosh committed:
Update and rename PetRepository.java to PetService.java

AvaMcintosh committed:
Add files via upload

AvaMcintosh committed:
Create Pet-README.md

Commit:
Created adopter

Commit:
Create adopter

Commit:
Resolved conflicts

hmcintos committed:
fixed up some errors and added in the staff portion of our microservices

hmcintos committed:
Create Staff-ReadME.md

AvaMcintosh committed:
Update init_shelter.sql

updating pet attributes to pet create table

hmcintos committed:
Update init_shelter.sql

hmcintos committed:
Update init_shelter.sql

AvaMcintosh committed:
Create test

Commit:
Adding basis for Web Design

Commit:
Finished base structure
Need to add to homepage, application, adopter, staff
Need to finish pet

hmcintos committed:
updated docker filer and init file to work with our database

hmcintos committed:
Merge branch 'main' of github.com:WSU-kduncan/cs3900-animalshelter

Commit:
this should push web design

Commit:
Merge branch 'main' of https://github.com/WSU-kduncan/cs3900-animalshelter

StarboardHook committed:
Set homepage, attempted made on Header, Footer, Navbar components but unable to get it to run.
reset project from default state.

Commit:
Created project

hmcintos committed:
Update staff.component.ts

hmcintos committed:
Update staff.component.ts

StarboardHook committed:
Set Footer to bottom of page, homepage set as default.

StarboardHook committed:
Merge branch 'main' of github.com:WSU-kduncan/cs3900-animalshelter

Commit:
i just wanna see what this is

hmcintos committed:
Updated staff code to be an integer instead of a string

hmcintos committed:
Updated the staff put mapping

hmcintos committed:
updated the path variables for put and delete mappings

StarboardHook committed:
@RequestMapping for Adoption, Application controllers handled.

StarboardHook committed:
Merge branch 'main' of github.com:WSU-kduncan/cs3900-animalshelter

Commit:
hope and pray

Commit:
Merge branch 'main' of https://github.com/WSU-kduncan/cs3900-animalshelter

hmcintos committed:
Updated the angular version and these were changes it made along the way

rjnicolai5 commented:
#Web Design Feedback

## Animal Shelter:
- Awesome, very well done so far.
- Realistically the ONLY thing I need to see at this point is http reqs working
- Good styles, reactive behavior, well organized
- Project / component structure is healthy, logical and good
- If I got to nitpick, I'd encourage more custom data models for things beyond the basic 'pet' model. (Adopters etc. If you have data objects, or something resembling one, give it its own model!)

hmcintos committed:
Updated the angular version and these were changes it made along the way

StarboardHook committed:
Accepted changes

Commit:
hope this works

Commit:
Merge branch 'main' of https://github.com/WSU-kduncan/cs3900-animalshelter

StarboardHook committed:
Merge branch 'main' of github.com:WSU-kduncan/cs3900-animalshelter

StarboardHook committed:
Fixed some issues pointed out by Microservices teams. All fixed points were marked as Resolved in Github

hmcintos committed:
fixed server.ts file to work with angular 19

hmcintos committed:
Merge branch 'main' of https://github.com/WSU-kduncan/cs3900-animalshelter

hmcintos committed:
updated pet form to match DB

hmcintos committed:
updated the pet form

caraz22 committed:
queries

caraz22 committed:
Merge branch 'main' of https://github.com/WSU-kduncan/cs3900-animalshelter

hmcintos committed:
Updated the pet form

hmcintos committed:
Merge branch 'main' of https://github.com/WSU-kduncan/cs3900-animalshelter

StarboardHook committed:
fixed placeholders for weight and age

StarboardHook committed:
Added findBySearch functionality to petService.java and petRepository.java

Commit:
Updated UI

Commit:
Merge branch 'main' of https://github.com/WSU-kduncan/cs3900-animalshelter

hmcintos committed:
Fixed the JSon and pet repository

hmcintos committed:
ITS ALIVE!!!!!!!!

hmcintos committed:
updated search query and launch.json

Commit:
Adding getPets http call still need to add url

caraz22 committed:
generated application service

Commit:
Added http request to pet.service.ts
Haven't tested to see if it works yet

Commit:
Merge branch 'main' of https://github.com/WSU-kduncan/cs3900-animalshelter

caraz22 committed:
used pet service as a template for application service

caraz22 committed:
deleted unnecessary imports

Commit:
Added service and model for adopters
Added http calls for adopters
changed pet id to pet_id in model and service

Commit:
Merge branch 'main' of https://github.com/WSU-kduncan/cs3900-animalshelter

Commit:
Changed pet_id back to id because it caused issues

caraz22 committed:
staff service complete

caraz22 committed:
Merge branch 'main' of https://github.com/WSU-kduncan/cs3900-animalshelter

hmcintos committed:
Changes post presentation for microservices

hmcintos committed:
Merge branch 'main' of https://github.com/WSU-kduncan/cs3900-animalshelter

hmcintos committed:
bringing microservices up to date with my repo

Commit:
Found the correct url
Tested get and add, doesn't work

Commit:
Merge branch 'main' of https://github.com/WSU-kduncan/cs3900-animalshelter

rjnicolai5 commented:
FE Grade Estimation: 95

- Conflicting Jest configurations do not allow unit tests to be run, please ensure you only have one jest config file
- Good work on getting HTTP requests up to snuff
- Great organization on files in general, I'd recommend moving your model and services folders one level deeper into the app folder.

StarboardHook committed:
Moving existing work to new branch

StarboardHook committed:
- Added search bar to Adopters, Applications and Pets pages.
- Added detailed listings for Applications.
- moved Status to be between buttons
- added Add function to applications page.

AvaMcintosh committed:
Update README.md

Updated first parts of READ.md for final project

AvaMcintosh committed:
Update README.md

Completed Final Project Read.me

AvaMcintosh committed:
Update README.md

fixed link

hmcintos committed:
Presentation for winsupply

StarboardHook committed:
Merging WebDesgin to Main.

StarboardHook committed:
Merge pull request #10 from WSU-kduncan/WebDesign

Web design
